samples/tools/autogenbench/pyproject.toml

```diff
@@ -5,7 +5,7 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "autogenbench"
 authors = [
-  { name="Autogen Team", email="auto-gen@outlook.com" },
+  { name="Autogen Team", email="autogen-contact@service.microsoft.com" },
```

Comment:
This is new. Who owns this?

Reply:
@randombet -- this new address for microsoft/autogen is owned by Microsoft.